### ISSUE TITLE
CmsKit - Not Found pages triggers infinite loop - ERR_TOO_MANY_REDIRECTS

### ISSUE THREAD

**Issue by enisn** [abpframework/abp#17279]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

 as an example, browser tries to load the following file `/__bundles/bootstrap.min.css.map` and infinite redirection loop starts. 
![image](https://github.com/abpframework/abp/assets/23705418/db0d59e9-52dc-4614-bd45-06e566e1f03d)

![image](https://github.com/abpframework/abp/assets/23705418/457d2d74-c80b-4744-a5e5-bbc9ad6ca5e2)

When I remove CmsKit Global Features;
```csharp
GlobalFeatureManager.Instance.Modules.CmsKit(cmsKit =>
{
    //cmsKit.EnableAll();
});
```
everything works as expected and there is no problem

### Version

7.3.1

### User Interface

Common (Default)

### Database Provider

None/Others

### Tiered or separate authentication server

None (Default)

### Operation System

Windows (Default)

**Comment by enisn** [comment]
Also, CmsKit tries to handle `/Error` page and redirection loop is triggered again

![image](https://github.com/abpframework/abp/assets/23705418/643a6c4a-f03f-44ff-9181-16064f5e8b08)

**Comment by enisn** [comment]
Found the problem:
https://github.com/abpframework/abp/blob/34835b5734b5cde81c9c8fb9599ae6fdde640471/modules/cms-kit/src/Volo.CmsKit.Public.Web/CmsKitPublicWebModule.cs#L93

The pattern above captures all paths that are not matched any route from routing, and also handles **(Not Found - 404)** too since it matches anything with min length 1.

## Workaround
Changing pages url prefix in **PreConfigureServices** prevents handling everyting

```csharp
public override void PreConfigureServices(ServiceConfigurationContext context)
{
    PageConsts.UrlPrefix = "pages/";
}
```

**Comment by enisn** [comment]
I think we can handle requests in a middleware instead registering all routes that starts with `/`

Like here:
https://github.com/volosoft/volo/blob/e5fae8331664a8e8596b499de02d0c6718a608a4/abp/cms-kit-pro/src/Volo.CmsKit.Pro.Public.Web/Middlewares/UrlShortingMiddleware.cs#L23

**Comment by enisn** [comment]
There is no quick fix without a breaking-change, 
So, I'm closing this issue since it will be solved with #17319 in the **8.0** milestone.
